src/main/resources/config.sample.yml

```diff
@@ -43,6 +43,9 @@ mail :
     # disabled = true will ignore the mail configuration and print to the command-line
     disabled : false
 
+    #specifies the mail address, that gets displayed as sender in moneytransfer mails
+    sendMail: pseipd@gmail.com
```

Comment:
maybe use moneytransferMailAdress, since sendmail is often the acronym for the MDA.